engineering/conventions-go.md

```diff
@@ -0,0 +1,50 @@
+
+# Go Development Conventions
+
+## Introduction
+
+This guide documents development conventions for Go at source{d}. Check [general conventions](conventions.md) for language-independent conventions that are also applicable for Go projects.
+
+## Supported Go Versions
+
+* Support latest two stable major versions of Go (e.g. 1.9.x and 1.10.x).
```

Comment:
Are these the versions used by default on continuous integration?
What about tip?

Reply:
tip, can't be a supported one, because is tip, what can be is on travis, to know if we will be ready.

Reply:
Some projects need to update their `.travis.yml`, but yes. `1.9.x` and `1.10.x` should be tested on continuous integration. We usually add `tip` too, but we set it with the Travis `allow_failures` flag on.

I reworded it to include it.